### PR TITLE
[feature/64/company-withdrawal-request] creates company withdrawal re…

### LINKED ISSUE #64: Company Withdrawal Request
Company should be able to make withdrawal request

### LINKED ISSUE #66: Cannot access user's wallet
The user query GetWalletById is expected to return the wallet details for the user instead it throws the following error when called, possibly because the table has not been dropped and recreated.

**Please treat asap**

"errors": [
    {
      "debugMessage": "SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation \"wallets\" does not exist\nLINE 1: select * from \"wallets\" where \"id\" = $1\n                      ^ (SQL: select * from \"wallets\" where \"id\" = dd8a3d55-df00-417d-983b-a502e1a14050)",
      "message": "Internal server error",
      "extensions": {
        "category": "internal"
      },

### PR DESCRIPTION
**Pull request by Oluwablin**
adds company withdrawal request with tests

### CODE REVIEW

File: graphql/models/Transaction.graphql [hunk 1]
```diff
@@ -32,13 +32,15 @@ enum TransactionType {
     BRANCH_FUND_DISBURSEMENT
     WALLET_PAYMENT
     WALLET_WITHDRAWAL
+    COMPANY_WITHDRAWAL
```
Comment (omadoyeabraham) [inline graphql/models/Transaction.graphql]: @Oluwablin @davidubanyi  

What is the difference between a COMPANY_WITHDRAWAL and a BRANCH_FUND_DISBURSEMENT?

File: graphql/models/UserProfile.graphql [hunk 2]
```diff
@@ -31,8 +31,10 @@ type CompanyBranch {
     id: ID!
     name: String!
     location: String
+    company_balance: Float!
```
Comment (omadoyeabraham) [inline graphql/models/UserProfile.graphql]: @Oluwablin 
This field should be renamed to "branch_balance", it's more specific that way.

**Review by omadoyeabraham — CHANGES_REQUESTED** [review]
@Oluwablin 

Please address the various comments made.

File: graphql/models/UserProfile.graphql (hunk 2 above)
Comment (Oluwablin) [reply graphql/models/UserProfile.graphql]: Okay, sir.

Comment (Oluwablin) [reply graphql/models/UserProfile.graphql]: > @Oluwablin
> This field should be renamed to "branch_balance", it's more specific that way.

@omadoyeabraham done

**Pull request closed without merging**